src/styles/components/_specification-block.scss

```diff
@@ -1,56 +1,23 @@
 .cmp-specifications-block {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
   gap: 3.75rem;
   line-height: 1.5625rem;
   letter-spacing: 0.02rem;
 
-  @media (min-width: $breakpoint-grid-large-screen-1115) {
-    display: grid;
-    column-gap: 5rem; // 80px /16
-    row-gap: 2.5rem; // 40px
-    grid-template-columns: 1fr 1fr;
-    grid-template-areas:
-      "keyboard semantic"
-      "focus focus"
-      "aria aria"
-      "additional .";
+  @supports (grid-template-rows: masonry) {
+    grid-template-rows: masonry;
   }
 
-  &__wrapper {
-    display: block;
+  @supports not (grid-template-rows: masonry) {
+    &__wrapper {
+      --row-start: auto;
+      --number-of-rows: 1;
+      --column-start: auto;
+      --number-of-columns: 1;
 
-    @media (min-width: $breakpoint-grid-large-screen-1115) {
-      margin: 0;
-
-      // focus expectations (1 column)
-      &:nth-child(1) {
-        grid-area: keyboard;
-      }
-
-      // semantic elements (1 column)
-      &:nth-child(2) {
-        grid-area: semantic;
-      }
-
-      // aria roles & attributes (full width)
-      &:nth-child(3) {
-        grid-area: aria;
-        max-width: 75ch;
-      }
-
-      // keyboard interactions (full-width)
-      &:nth-child(4) {
-        grid-area: focus;
-        max-width: 75ch;
-      }
-
-      // additional resources (1 column)
-      &:last-child {
-        li {
-          padding: 0.25rem 0; // top&bottom: 4px/16
-        }
-      }
+      grid-row: var(--row-start, 1) / span var(--number-of-rows, 1);
+      grid-column: var(--column-start, 1) / span var(--number-of-columns, 1);
```

Comment:
One of the styles that was removed was `max-width: 75ch`, which I think would be good to add back in, especially if we decide to make certain SpecificationBlocks span 2 columns.

Reply:
Thanks, I rolled that change into this commit: https://github.com/sparkbox/accessible-components/pull/99/commits/11fd0de23ba5880ce3bfd2135e256cc14a21ce39